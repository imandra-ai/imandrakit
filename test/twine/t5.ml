type t = unit [@@deriving twine]

let m : string = Imandrakit_twine.Encode.to_string to_twine ()
let () = Format.printf "m:@.%s@." (Hex.hexdump_s @@ Hex.of_string m)
let () = Format.printf "size of empty msg is %d@." (String.length m)
