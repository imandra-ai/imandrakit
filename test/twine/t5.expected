size of empty msg is 7
